rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -311,7 +353,14 @@ def mock_requests_get(url: str, timeout: int = 0):  # pylint: disable=unused-arg
             data_dir=data_dir,
             tolerance_for_amount_check=tolerance_for_amount_check,
         )
-        assert mock_get.call_count == 1  # queried again because it was cached 12 hours ago
+        # diva CSV and aave JSON were queried again because their hashes were updated
+        assert mock_get.call_count == 3  # index queried again because it was cached 12 hours ago
+
+    # new CSV hashes are saved in the DB
+    with globaldb.conn.read_ctx() as cursor:
+        assert cursor.execute(
+            'SELECT value FROM unique_cache WHERE key=?', ('AIRDROPS_HASHdiva',),
+        ).fetchone()[0] == 'updated_hash'
```

Comment:
Also in discord you wrote:

> the other was the case for user who have recently downloaded index.json. It's cached for 12 hours, so they were continuing with old index.json with new code. Which was failing because csv_hash is not present in the old index json

Is this case tested anywhere?  I am not sure the implementation is correct